tools/run_cargo_fmt.sh

```diff
@@ -86,6 +86,15 @@ done
 rm xx00
 printf "\rFormatting complete. %-$((39))s\n" ""
 
+# Check for tab characters in Rust source files that haven't been
+# removed by rustfmt
+RUST_FILES_WITH_TABS="$(git grep --files-with-matches $'\t' -- '*.rs' || grep -lr --include '*.rs' $'\t' || true)"
```

Comment:
Unfortunately this gives me `grep: warning: recursive search of stdin` on mac.

Reply:
Just need to add a directory to start from at the end of the command, which should work on *nix too I believe:

```
$ grep -lr --include '*.rs' $'\t' .
./boards/components/src/led.rs
./boards/components/src/led_matrix.rs
./capsules/src/l3gd20.rs
./capsules/src/lsm303agr.rs
./capsules/src/lsm303dlhc.rs
./capsules/src/mlx90614.rs
./chips/stm32f303xc/src/i2c.rs
./kernel/src/debug.rs
```

Reply:
Yeah, that works, can confirm on my new FreeBSD development VM :smile:. Will update accordingly, thanks!

Reply:
Fixed.